setup.py

```diff
@@ -115,28 +115,45 @@ def run(self):
         version=version,
         author='Edgar Riba',
         author_email='contact@kornia.org',
-        url='https://github.com/kornia/kornia',
+        url='https://www.kornia.org',
+        download_url='https://github.com/kornia/kornia',
+        license='Apache License 2.0',
         description='Open Source Differentiable Computer Vision Library for PyTorch',
         long_description=long_description,
         long_description_content_type='text/markdown',
-        license='Apache License 2.0',
         python_requires='>=3.6',
-        # Test
         setup_requires=['pytest-runner'],
         tests_require=['pytest'],
-        # Package info
         packages=find_packages(exclude=('docs', 'test', 'examples')),
         package_data={"kornia": ["py.typed"]},
         zip_safe=True,
         install_requires=requirements,
+        keywords=['computer vision', 'deep learning', 'pytorch']
+        project_urls={
+            "Bug Tracker": "https://github.com/kornia/kornia/issues",
+            "Documentation": "https://kornia.readthedocs.io/en/latest/",
+            "Source Code": "https://github.com/kornia/kornia",
+        },
         classifiers=[
+            'Environment :: Console',
+            'Natural Language :: English',
+            # How mature is this project? Common values are
+            #   3 - Alpha, 4 - Beta, 5 - Production/Stable
+            'Development Status :: 4 - Beta',
+            # Indicate who your project is intended for
             'Intended Audience :: Developers',
             'Intended Audience :: Education',
             'Intended Audience :: Science/Research',
-            'Operating System :: POSIX :: Linux',
-            'Programming Language :: Python :: 3 :: Only',
-            'License :: OSI Approved :: Apache Software License',
+            'Topic :: Scientific/Engineering :: Artificial Intelligence',
             'Topic :: Scientific/Engineering :: Image Recognition',
```

Comment:
Is it better to be computer vision than image recognition?

Also, probably similar to the keywords?

```suggestion
            'Topic :: Scientific/Engineering :: Computer Vision',
```

Reply:
It’s a predefined list. See: https://pypi.org/classifiers/